source/gui/__init__.py

```diff
@@ -348,11 +348,17 @@ def onRunCOMRegistrationFixesCommand(self, evt):
 			log.error("Could not execute fixCOMRegistrations command",exc_info=True) 
 		progressDialog.done()
 		del progressDialog
-		# Translators: The message displayed when the COM Registration Fixing tool completes.
-		gui.messageBox(_("COM Registration Fixing tool complete"),
+		# We get an F405 here because "gui may be undefined".
+		gui.messageBox(  # noqa F405

```

Comment:
Avoid NOQA. Since `messageBox` is defined in this file you can just do `messageBox(blah)`

Reply:
Done. That's what I get for sleepworking.